docs/flashbots-mev-share/searchers/getting-started.mdx

```diff
@@ -9,20 +9,20 @@ To start searching on MEV-Share, you will first need to connect to a MEV-Share N
 
 ## Connect to MEV-Share Node
 
-Flashbots runs a MEV-Share Node on Ethereum mainnet. The MEV-Share Node has endpoints to receive transactions and bundles, as well as an SSE event stream endpoint which dispatches pending events/transactions to searchers.
+Flashbots operates an MEV-Share Node on the Ethereum mainnet. This node provides endpoints for user transactions and searcher bundles. Additionally, it offers an SSE event stream endpoint, broadcasting pending events/transactions to searchers.
 
-The simplest way to connect to the Flashbots MEV-Share Node is to use a [client library](/flashbots-auction/searchers/libraries/mev-share-clients). For this guide, we'll refer to [mev-share-client-ts](https://github.com/flashbots/mev-share-client-ts).
+The simplest way to connect to the Flashbots MEV-Share Node is to use a [client library](/flashbots-auction/searchers/libraries/mev-share-clients). For this guide, we'll refer to [mev-share-client-ts](https://github.com/flashbots/mev-share-client-ts). Several other clients are being developed.
```

Comment:
I don't think we need this line; the link to client library will show that there are others.